docker-swarm/.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=autopilotmysql
```

Comment:
I'm happy to drop these `.env` files, but they make our container names default to `autopilotmysql_XXX_N` instead of `dockerswarm_XXX_N` (due to the subdirectory name).